iOS components (#2375)
clerk/clerk-docs: docs/references/ios/auth-view.mdx

`AuthView` provides a complete authentication experience with support for multiple sign-in methods, sign-up flows, multi-factor authentication, password reset, and account recovery. The view can be configured for different authentication modes and automatically handles navigation between authentication steps.

## Parameters

<Properties>
  - `mode`
  - `AuthView.Mode`

  The authentication mode that determines which flows are available to the user. Accepts the following values:

  - `.signInOrUp`: Automatically determines whether to sign users in or sign them up based on whether they already have an account. This is the default mode that provides seamless authentication without requiring users to choose between sign-in and sign-up.
  - `.signIn`: Restricts the interface to sign-in flows only. Users can only authenticate with existing accounts.
  - `.signUp`: Restricts the interface to sign-up flows only. Users can only create new accounts.

  Defaults to `.signInOrUp`.

  ---

  - `isDismissable`
  - `Bool`

>   Whether the view can be dismissed by the user. When `true`, a dismiss button appears and the view automatically dismisses on successful authentication. When `false`, no dismiss button is shown. Defaults to `true`.

Reviewer: ```suggestion
  Whether the view can be dismissed by the user. When `true`, a dismiss button appears and the view closes automatically after successful authentication. When `false`, no dismiss button is shown. Defaults to `true`.
```
